### PR TITLE
Completed registration

### PR DESCRIPTION
**Pull request by erndawg22**
### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [ ] Added a telegram handle of an **individual** and not that of a group or channel.
- [ ] Added a social media account with activity.
- [ ] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [ ] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [ ] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -4705,5 +4705,19 @@
...
+    "ticker": "LSSP",                                                       
+    "poolId": "01fbe26f514707ff29c38aa597ad8ff86a1572e760858d0e7ae4fc32", 
+    "social": {                                                            
+      "twitter": "Lonestarstaking",
+      "website": "https://lonestarstaking.net",
+      "facebook": "",  
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "erndawg22",
```
Comment (Bez625) [inline registry.json:4719]: I cant find this user on telegram. Please can you confirm?

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by erndawg22** [comment]
Sorry, “erndawg”……. Only…  On Tue, Apr 26, 2022 at 8:58 AM, Chris Berry ***@***.***> wrote:  
@Bez625 requested changes on this pull request.



In registry.json:
> +    },
+    "telegram": "erndawg22",

I cant find this user on telegram. Please can you confirm?

File: registry.json (hunk 1 above)
Comment (Bez625) [inline registry.json:4719]:
```suggestion
    "telegram": "erndawg",                                          
```

**Comment by erndawg22** [comment]
Thank you..  On Tue, Apr 26, 2022 at 9:07 AM, Chris Berry ***@***.***> wrote:  
@Bez625 commented on this pull request.



In registry.json:
> @@ -4705,5 +4705,19 @@
     "telegram": "RickCardo",
     "operatorName": "Rick Carstens",
     "github": "Brolloks"
+    },
+ {
+    "ticker": "LSSP",
+    "poolId": "01fbe26f514707ff29c38aa597ad8ff86a1572e760858d0e7ae4fc32",
+    "social": {
+      "twitter": "Lonestarstaking",
+      "website": "https://lonestarstaking.net",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "erndawg22",

⬇️ Suggested change
-    "telegram": "erndawg22",
+    "telegram": "erndawg",

**Pull request merged**